ext/opcache/Optimizer/zend_func_info.c

```diff
@@ -242,74 +226,39 @@ static const func_info_t func_infos[] = {
 	F1("shell_exec",                   MAY_BE_NULL | MAY_BE_FALSE | MAY_BE_STRING),
 #ifdef PHP_CAN_SUPPORT_PROC_OPEN
 	F1("proc_open",                    MAY_BE_FALSE | MAY_BE_RESOURCE),
-	F0("proc_terminate",               MAY_BE_FALSE | MAY_BE_TRUE),
 	F1("proc_get_status",              MAY_BE_ARRAY | MAY_BE_ARRAY_KEY_STRING | MAY_BE_ARRAY_OF_FALSE | MAY_BE_ARRAY_OF_TRUE | MAY_BE_ARRAY_OF_LONG | MAY_BE_ARRAY_OF_STRING),
 #endif
 #ifdef HAVE_NICE
 	F0("proc_nice",                    MAY_BE_FALSE | MAY_BE_TRUE),
 #endif
 	F1("random_bytes",                 MAY_BE_STRING),
-	F1("random_int",                   MAY_BE_LONG),
```

Comment:
Shouldn't this be kept because it's F1 ?

Reply:
That was a (harmless) mistake in the data. MAY_BE_LONG cannot be RC1.

Reply:
Okay then :)

Reply:
I added the entry for "random_int" before being familiar with the meanings in September (basing it on random_bytes); the Reflection types were added in October.